packages/react/src/Accordion/Accordion.module.css

```diff
@@ -252,10 +252,8 @@ details[open] > .Accordion__summary.Accordion__summary--default .Accordion__summ
   color: var(--brand-InlineLink-color-rest);
   position: relative;
   text-decoration: underline;
-  text-decoration-color: var(--brand-InlineLink-color-rest);
```

Comment:
@stamat i think removing this has caused a visual regression. As you're trying to remove the transition, i'm guessing it's unintentional. Could we restore it please?